generic3g/GenericGridComp.F90

```diff
@@ -170,6 +170,8 @@ recursive subroutine initialize(gridcomp, importState, exportState, clock, rc)
          call outer_meta%initialize_realize(_RC)
 !#      case (GENERIC_INIT_RESTORE)
 !#         call outer_meta%initialize_realize(_RC)
+      case (GENERIC_INIT_READ_RESTART)
```

Comment:
Same comment as above about deleting preceding lines.  (And these have a bad cut-and-paste-error as well.)

Reply:
ok.